### DISCUSSION TITLE
Audiokit v2.2 A247 no Sound

### DISCUSSION THREAD

**Discussion by wmnbg** [pschatzmann/arduino-audiokit#78] [Q&A] [answered]
Hi,

i have a Audiokit v2.2 A247 and I tried several Sketches. The board is recognized as Board 5 and it seems,  the program is running correctly.
It seems, that something mutes the output. Both outputs doesn't work (Headphone and Speaker)
I didnt find anything about this board version in the web. Does anybody use the same board?

The dipswitches are 1,2,3=ON     4,5=OFF

Last I tried the Webradio of Phil _streams-url_mp3-audiokit.ino_ also there is no sound
Would be great if anybody have a Tip

thx

Here the Serial Output at Start

[I] AudioTypes.h : 78 - sample_rate: 44100
[I] AudioTypes.h : 79 - channels: 2
[I] AudioTypes.h : 80 - bits_per_sample: 16
[I] I2SConfig.h : 106 - rx/tx mode: TX_MODE
[I] I2SConfig.h : 107 - port_no: 0
[I] I2SConfig.h : 108 - is_master: Master
[I] I2SConfig.h : 109 - sample rate: 44100
[I] I2SConfig.h : 110 - bits per sample: 16
[I] I2SConfig.h : 111 - number of channels: 2
[I] I2SConfig.h : 112 - i2s_format: I2S_STD_FORMAT
[I] I2SConfig.h : 114 - auto_clear: true
[I] I2SConfig.h : 116 - use_apll: true
[I] I2SConfig.h : 121 - buffer_count:6
[I] I2SConfig.h : 122 - buffer_size:512
[I] I2SConfig.h : 126 - pin_bck: 27
[I] I2SConfig.h : 128 - pin_ws: 25
[I] I2SConfig.h : 130 - pin_data: -1515870811
[W] AudioKit.h : 58 - sd_active = true
[I] I2SConfig.h : 106 - rx/tx mode: TX_MODE
[I] I2SConfig.h : 107 - port_no: 0
[I] I2SConfig.h : 108 - is_master: Master
[I] I2SConfig.h : 109 - sample rate: 44100
[I] I2SConfig.h : 110 - bits per sample: 16
[I] I2SConfig.h : 111 - number of channels: 2
[I] I2SConfig.h : 112 - i2s_format: I2S_STD_FORMAT
[I] I2SConfig.h : 114 - auto_clear: true
[I] I2SConfig.h : 116 - use_apll: true
[I] I2SConfig.h : 121 - buffer_count:6
[I] I2SConfig.h : 122 - buffer_size:512
[I] I2SConfig.h : 126 - pin_bck: 27
[I] I2SConfig.h : 128 - pin_ws: 25
[I] I2SConfig.h : 130 - pin_data: 26
[I] AudioKit.h : 591 - void audio_tools::AudioKitStream::setupActions()
[W] AudioKit.h : 598 - Mode Button ignored because of conflict: 13 
[I] AudioActions.h : 42 - ActionLogic::add pin: 39 / logic: 2
[I] AudioActions.h : 55 - pin 39 -> INPUT
[I] AudioKit.h : 341 - void audio_tools::AudioKitStream::addAction(int, void (*)(bool, int, void*), void*)
[I] AudioActions.h : 42 - ActionLogic::add pin: 18 / logic: 0
[I] AudioActions.h : 52 - pin 18 -> INPUT_PULLUP
[I] AudioKit.h : 341 - void audio_tools::AudioKitStream::addAction(int, void (*)(bool, int, void*), void*)
[I] AudioActions.h : 42 - ActionLogic::add pin: 5 / logic: 0
[I] AudioActions.h : 52 - pin 5 -> INPUT_PULLUP
[I] URLStream.h : 89 - virtual bool audio_tools::URLStream::begin(const char*, const char*, MethodID, const char*, const char*): http://stream.srg-ssr.ch/m/rsj/mp3_128
[I] Url.h : 74 - Url::parse
[I] Url.h : 110 - url->http://stream.srg-ssr.ch/m/rsj/mp3_128
[I] Url.h : 111 - host->stream.srg-ssr.ch
[I] Url.h : 112 - protocol->http
[I] Url.h : 113 - path->/m/rsj/mp3_128
[I] Url.h : 114 - port->80
[I] URLStream.h : 325 - void audio_tools::URLStream::login()
.....
[I] URLStream.h : 300 - WiFiClient
[I] HttpRequest.h : 227 - process connecting to host stream.srg-ssr.ch port 80
[I] HttpRequest.h : 213 - connected 1 timeout 60000
[I] HttpRequest.h : 238 - Free heap: 220956
[I] HttpHeader.h : 361 - HttpRequestHeader::setValues - path: /m/rsj/mp3_128
[I] HttpHeader.h : 127 - Content-Length 0
[I] HttpHeader.h : 258 - HttpHeader::write
[I] HttpHeader.h : 370 - HttpRequestHeader::write1stLine
[I] HttpHeader.h : 385 - -> GET /m/rsj/mp3_128 HTTP/1.1
[I] HttpHeader.h : 193 -  -> Content-Length: 0 
[I] HttpHeader.h : 193 -  -> Host: stream.srg-ssr.ch 
[I] HttpHeader.h : 193 -  -> Connection: close 
[I] HttpHeader.h : 193 -  -> Accept: audio/mp3 
[I] HttpHeader.h : 301 -  -> <CR LF> 
[I] HttpRequest.h : 275 - Request written ... waiting for reply
[I] HttpHeader.h : 226 - HttpHeader::read
[W] HttpHeader.h : 233 - Waiting for data...
[I] HttpHeader.h : 163 - HttpHeader::readLine -> HTTP/1.1 200 OK
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Content-Type: audio/mpeg
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Date: Thu, 27 Apr 2023 21:03:33 GMT
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-br:128
[I] HttpHeader.h : 163 - HttpHeader::readLine -> ice-audio-info: bitrate=128
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-br:128
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-description:SwissJazz_MP3@128Kbps
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-genre:Entertainment
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-mimetype:audio/mpeg
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-name:/SwissJazz_Room1
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-pub:0
[I] HttpHeader.h : 163 - HttpHeader::readLine -> icy-url:www.srgssr.ch
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Server: Icecast 2.4.0-kh10
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Cache-Control: no-cache, no-store
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Access-Control-Allow-Origin: *
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Access-Control-Allow-Headers: Origin, Accept, X-Requested-With, Content-Type
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Access-Control-Allow-Methods: GET, OPTIONS, HEAD
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Connection: Close
[I] HttpHeader.h : 163 - HttpHeader::readLine -> Expires: Mon, 26 Jul 1997 05:00:00 GMT
[I] HttpHeader.h : 163 - HttpHeader::readLine -> 
[W] HttpRequest.h : 174 - no CONTENT_LENGTH found in reply
[I] URLStream.h : 110 - size: 0
[I] URLStream.h : 341 - virtual bool audio_tools::URLStream::waitForData()
[I] URLStream.h : 353 - available: 5177
[I] StreamCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops
[I] AudioKit.h : 246 - virtual void audio_tools::AudioKitStream::setAudioInfo(audio_tools::AudioInfo)
[I] I2SConfig.h : 106 - rx/tx mode: TX_MODE
[I] I2SConfig.h : 107 - port_no: 0
[I] I2SConfig.h : 108 - is_master: Master
[I] I2SConfig.h : 109 - sample rate: 48000
[I] I2SConfig.h : 110 - bits per sample: 16
[I] I2SConfig.h : 111 - number of channels: 2
[I] I2SConfig.h : 112 - i2s_format: I2S_STD_FORMAT
[I] I2SConfig.h : 114 - auto_clear: true
[I] I2SConfig.h : 116 - use_apll: true
[I] I2SConfig.h : 121 - buffer_count:6
[I] I2SConfig.h : 122 - buffer_size:512
[I] I2SConfig.h : 126 - pin_bck: 27
[I] I2SConfig.h : 128 - pin_ws: 25
[I] I2SConfig.h : 130 - pin_data: 26
[I] I2SStream.h : 74 - virtual void audio_tools::I2SStream::setAudioInfo(audio_tools::AudioInfo)
[I] AudioTypes.h : 78 - sample_rate: 48000
[I] AudioTypes.h : 79 - channels: 2
[I] AudioTypes.h : 80 - bits_per_sample: 16
[I] I2SConfig.h : 106 - rx/tx mode: TX_MODE
[I] I2SConfig.h : 107 - port_no: 0
[I] I2SConfig.h : 108 - is_master: Master
[I] I2SConfig.h : 109 - sample rate: 48000
[I] I2SConfig.h : 110 - bits per sample: 16
[I] I2SConfig.h : 111 - number of channels: 2
[I] I2SConfig.h : 112 - i2s_format: I2S_STD_FORMAT
[I] I2SConfig.h : 114 - auto_clear: true
[I] I2SConfig.h : 116 - use_apll: true
[I] I2SConfig.h : 121 - buffer_count:6
[I] I2SConfig.h : 122 - buffer_size:512
[I] I2SConfig.h : 126 - pin_bck: 27
[I] I2SConfig.h : 128 - pin_ws: 25
[I] I2SConfig.h : 130 - pin_data: 26
[I] I2SConfig.h : 106 - rx/tx mode: TX_MODE
[I] I2SConfig.h : 107 - port_no: 0
[I] I2SConfig.h : 108 - is_master: Master
[I] I2SConfig.h : 109 - sample rate: 48000
[I] I2SConfig.h : 110 - bits per sample: 16
[I] I2SConfig.h : 111 - number of channels: 2
[I] I2SConfig.h : 112 - i2s_format: I2S_STD_FORMAT
[I] I2SConfig.h : 114 - auto_clear: true
[I] I2SConfig.h : 116 - use_apll: true
[I] I2SConfig.h : 121 - buffer_count:6
[I] I2SConfig.h : 122 - buffer_size:512
[I] I2SConfig.h : 126 - pin_bck: 27
[I] I2SConfig.h : 128 - pin_ws: 25
[I] I2SConfig.h : 130 - pin_data: 26
[I] StreamCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops
[I] StreamCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops
[I] StreamCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops
[I] StreamCopy.h : 136 - StreamCopy::copy 1024 -> 1024 -> 1024 bytes - in 1 hops

**Comment by pschatzmann** [comment]
Check in the Boards manager, that you are using an actual ESP32 version!
The current version is 2.0.8!

**Comment by wmnbg** [comment, marked as answer]
Thx Phil, 
that was the Problem.
After Update I tried the webradio and it works, also the streams-synthbasic-audiokit.ino
